src/jshint.js

```diff
@@ -132,17 +132,13 @@ var JSHINT = (function() {
     return true;
   }
 
-  function isReserved(token) {
+  function isReserved(context, token) {
```

Comment:
I was curious about `context` being added as the first parameter in the list, so I went looking for other examples and discovered that `function expression(rbp, context)` is the only place where the `context` bitfield is "not the first" parameter. I wonder if that function should be updated to match the signature, ie. `context` first?

Reply:
I would expect that update to occur in a different changeset.

Reply:
Totally!